pkg/state/types.go

```diff
@@ -565,14 +566,14 @@ func (p1 *Plugin) Console() string {
 // Equal returns true if r1 and r2 are equal.
 // TODO add compare array without position
 func (p1 *Plugin) Equal(p2 *Plugin) bool {
```

Comment:
For plugin, this function didn't seem to be used directly anywhere.
However, it makes sense to expect schema as org here as well - will make that change in next pass.

Reply:
As far as naming convention goes, any "options" should be passed as args in EqualWithOpts function only. So, we should keep this one as it is.